rbac/migration_tool/migrate.py

```diff
@@ -104,43 +104,87 @@ def migrate_role(role: Role, write_db: bool):
     v1_role = dataclasses.replace(v1_role, groups=frozenset(groups))
 
     # This is where we wire in the implementation we're using into the Migrator
-    v1_to_v2_mapping = shared_system_role_replicated_role_bindings_v1_to_v2_mapping
-    permissioned_role_migrator = Migrator(v1_to_v2_mapping)
-    v2_roles = [v2_role for v2_role in permissioned_role_migrator.migrate_v1_roles(v1_role)]
-    relationships = spicedb_relationships(frozenset(v2_roles))
+    v2_roles = [v2_role for v2_role in role_v1_to_v2_mapping(v1_role, root_workspace, default_workspace)]
+    relationships = spicedb_relationships(frozenset(v2_roles), root_workspace)
     for rel in relationships:
         logger.info(stringify_spicedb_relationship(rel))
     if write_db:
         write_relationships(relationships)
 
 
-def migrate_roles_for_tenant(tenant: Tenant, app_list: list, write_db: bool):
-    """Migrate all roles for a given tenant."""
+def migrate_workspace(tenant: Tenant, write_db: bool):
+    """Migrate a workspace from v1 to v2."""
+    root_workspace = Workspace.objects.create(name="root", description="Root workspace", tenant=tenant)
+    # Org id represents the default workspace for now
+    relationships = [
+        create_relationship("workspace", tenant.org_id, "workspace", str(root_workspace.uuid), "parent"),
+    ]
+    if write_db:
+        write_relationships(relationships)
+    return root_workspace, tenant.org_id
+
+
+def migrate_users(tenant: Tenant, write_db: bool):
+    """Write users relationship to tenant."""
+    relationships = [
+        create_relationship("user", str(principal.uuid), "tenant", str(tenant.org_id), "orgization")
```

Comment:
I think we want to:
- update the relation to `member`
- swap the `resource` and `subject` ordering

```suggestion
        create_relationship("tenant", str(tenant.org_id), "user", str(principal.uuid), "member")
```